### DISCUSSION TITLE
Void Linux port

### DISCUSSION THREAD

**Discussion by zen0bit** [nwg-piotr/nwg-shell#308] [Porting for other distros]
# I would love to use nwg-shell on void linux.
Already requested https://github.com/void-linux/void-packages/issues/46643

Trying to packaging all missed parts. Some already there.

This will track my progress, and maybe someone help me...

### Already in void:
- [x] azote
- [x] nwg-launchers
- [x] nwg-look

### nwg-shell parts:
- [x] nwg-displays
- [x] nwg-drawer
- [x] nwg-menu
- [x] nwg-bar
- [x] nwg-clipman
- [x] nwg-icon-picker
- [x] nwg-dock
- [x] nwg-dock-hyprland
- [x] nwg-readme-browser
- [x] nwg-panel

### Dependencies already in void:
- [x] gnome-themes-extra
- [x] grim
- [x] gtklock
- [x] ImageMagick
- [x] jq
- [x] network-manager-applet
- [x] papirus-icon-theme
- [x] playerctl
- [x] polkit-gnome
- [x] python3-yaml
- [x] slurp
- [x] swappy
- [x] sway
- [x] swaybg
- [x] swayidle
- [x] swaylock
- [x] SwayNotificationCenter (swaync)
- [x] dejavu-fonts-ttf
- [x] wl-clipboard
- [x] wl-sunset
- [x] xorg-server-xwayland

### Not sure if everything is here, or need separate package...
- [x] python3-dasbus
- [ ] python3-geopy
- [ ] libgeographic
- [ ] gotk3
- [ ] nwg-shell-config
- [x] libayatana-appindicator ? Could be used instead of libappindicator-gtk3 (in AUR)

### Hyprland:
Can't be (yet) included
- [x] but template here: https://github.com/Makrennel/hyprland-void

# And finally:
- [x] nwg-shell (Still missing nwg-shell-config)

Am I missed something?

**Comment by nwg-piotr** [comment]
Looks good to me. Let me know in case some action on my side is required.

**Comment by zen0bit** [comment]
I am strugling with building nwg-shell-config. that part which installs autotiling and other executables...

Also not solved python3-geopy/python3-geographiclib/libgeographic

Not sure which is really needed, whats don't.

nwg-shells parts looks like working all well. With exception of nwg-shell-config.
 I don't see any executable which should be provided by this package in my system.

**Comment by zen0bit** [comment]
Build falling https://github.com/void-linux/void-packages/actions/runs/8518416132/job/23330532049?pr=49664 :sob:
